sklearn/decomposition/tests/test_pca.py

```diff
@@ -349,3 +349,11 @@ def test_pca_score3():
         ll[k] = pca.score(Xt)
 
     assert_true(ll.argmax() == 1)
+
+def test_get_feature_names():
+    X = np.array([[-1, -1], [-2, -1], [-3, -2], [1, 1], [2, 1], [3, 2]])
+    pca = PCA(n_components=2)
+    pca.fit(X)
+    assert_equal(pca.get_feature_names(), ['pc0', 'pc1'])
+    assert_equal(pca.get_feature_names(full_output=True),
+        ['0.838 * x0 + 0.545 * x1', '0.545 * x0 - 0.838 * x1'])
```

Comment:
Probably should construct a test in which one of the components has a negative sign in the first coefficient.